internal/provider/team_data_source_test.go

```diff
@@ -35,7 +35,7 @@ func TestAccTeamDataSource(t *testing.T) {
 	stub := TestTeamDataSource{
 		Tag: "test-team",
 		Members: []cortex.TeamMember{
-			{Name: "Test User", Email: "test@example.com"},
+			{Name: "Test", Email: "test@cortex.io"},
```

Comment:
I see `Test Member` from a quick search -- where does `Test` come from? Does the name matter?

Reply:
I don't think the name matters here. 
I didn't realize we have a shared acceptance testing DB 🤔